### PR TITLE
Feat/changed application architecture, changed styles system

### PR DESCRIPTION
**Pull request by makarofalex3**
Сhanged application architecture:
- separated the components into separate layers
  - pages folder for use in a router
  - routes folder for different routes
  - ui folder for reuse parameters of components
  - components folder for reuse of components

-  in dockerfile 
   -  add tsconfig to resolve ts and absolute paths
   -  replaced npm to yarn

### CODE REVIEW

**Comment by vkomodey** [comment]
@makarofalex3 CI doesn't work. Need a fix

**Comment by vkomodey** [comment]
Please, add PR description with detailed explanation what(and why) this was implemented as well

**Review by vkomodey — CHANGES_REQUESTED** [review]
You updated yarn. Need to update instructions in the README file

File: src/queries/index.ts [hunk 1]
```diff
@@ -24,3 +27,64 @@ export const ESCROWFACTORY_COUNT = gql`
     }
   }
 `;
+
+export const IS_EVENT = gql`
```
[the remark this answers is not in the document]
Comment (makarofalex3) [reply src/queries/index.ts]: fixed

File: package.json [hunk 2]
```diff
@@ -21,25 +21,33 @@
...
+    "react-router-dom": "^6.3.0",
     "react-scripts": "4.0.3",
     "serve": "^13.0.2",
     "typescript": "^4.7.2",
+    "react-window": "^1.8.7",
     "web-vitals": "^2.1.4",
     "web3": "^1.7.1"
   },
   "devDependencies": {
+    "@types/react-window": "^1.8.5",
```
[the remark this answers is not in the document]
Comment (makarofalex3) [reply package.json]: fixed

File: Dockerfile [hunk 3]
```diff
@@ -2,15 +2,16 @@ FROM node:16.13-buster
 
 WORKDIR /usr/src/app
 
-COPY package.json package-lock.json ./
+COPY package.json yarn.lock ./
+COPY tsconfig.json ./
```
[the remark this answers is not in the document]
Comment (makarofalex3) [reply Dockerfile]: fixed

**Review by vkomodey — CHANGES_REQUESTED** [review]
The layout is broken for the last block. Compare current dashboard and yours
image
image

**Pull request merged**